eventsource/resources/last-event-id.py

```diff
@@ -1,11 +1,9 @@
-from six import ensure_str
-
 def main(request, response):
-  response.headers.set("Content-Type", "text/event-stream")
+  response.headers.set(b"Content-Type", b"text/event-stream")
 
-  last_event_id = ensure_str(request.headers.get("Last-Event-ID", ""))
+  last_event_id = request.headers.get(b"Last-Event-ID", b"")
   if last_event_id:
-    return "data: " + last_event_id + "\n\n"
+    return b"data: " + last_event_id + b"\n\n"
   else:
-    idvalue = request.GET.first("idvalue", u"\u2026")
-    return "id: " + idvalue + "\nretry: 200\ndata: hello\n\n"
+    idvalue = request.GET.first(b"idvalue", u"\u2026".encode("utf-8"))
```

Comment:
Why not isomorphic_encode?

Reply:
This is similar to the discussion at 
https://github.com/web-platform-tests/wpt/pull/24091#discussion_r438367986